FloconDesktop/data/remote/src/commonMain/kotlin/com/flocon/data/remote/network/datasource/NetworkRemoteDataSourceImpl.kt

```diff
@@ -51,7 +50,7 @@ class NetworkRemoteDataSourceImpl(
             ?.let(FloconNetworkCallIdDataModel::toDomain)
     }
 
-    override fun getResponseData(message: FloconIncomingMessageDomainModel): FloconNetworkResponseOnlyDomainModel? {
+    override fun getResponseData(message: FloconIncomingMessageDomainModel): FloconNetworkResponseDomainModel? {
```

Comment:
please use FloconNetworkResponseOnlyDomainModel, I did it it 2 times already, stop reverting it

Reply:
I tink you did a wrong rebase